apps/webcert/src/store/certificate/certificateActions.ts

```diff
@@ -396,7 +396,7 @@ export const copyCertificateSuccess = createAction<CopyCertificateSuccess>(COPY_
 
 export const copyCertificateCompleted = createAction(COPY_CERTIFICATE_COMPLETED)
 
-export const validateCertificate = createAction<Certificate>(VALIDATE_CERTIFICATE)
```

Comment:
`handleValidateCertificate` uses `getState()` to get the certificate. Passing the certificate to the function is not necessary.